.gitignore

```diff
@@ -108,3 +108,6 @@ dist
 /svelte.config-dist.js
 /docs-svelte-kit/shim/eslint.mjs
 /docs-svelte-kit/shim/assert.mjs
+
+# Intellij IDES
+.idea
```

Comment:
Personally speaking, `IDE` things should be ignored with global `gitignore` file on your own machine.